tilavarauspalvelu/api/graphql/mutations.py

```diff
@@ -130,6 +130,7 @@
     "RestoreAllApplicationOptionsMutation",
     "RestoreAllSectionOptionsMutation",
     "SetApplicationRoundHandledMutation",
+    "SetApplicationRoundResultsSentMutation",
```

Comment:
IMO These should be `ApplicationRoundSetResultsSentMutation` (for better alphabetical order) 🤓

Reply:
No gonna change them since that changes the GraphQL typing which would make this a breaking change.